app/views/user_mailer/reset_password.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Hi <%= @user.first_name %></h1>
+<p>Click on this <%= link_to 'link',  forgot_password_edit_url(id: @user.id, token: @user.reset_token) %> to change your password</p>
```

Comment:
create this method on user to get forgot link url

Reply:
Done